config/canonical.json

```diff
@@ -33568,11 +33568,10 @@
   },
   "tourism/hotel|Kyriad": {
     "count": 76,
-    "countryCodes": ["fr"],
     "tags": {
       "brand": "Kyriad",
-      "brand:wikidata": "Q529212",
-      "brand:wikipedia": "fr:Louvre Hotels Group",
+      "brand:wikidata": "Q11751808",
+      "brand:wikipedia": "pl:Kyriad",
```

Comment:
Why do you link to the Polish wikipedia page if this hotel does only exist in France?

Reply:
See also https://overpass-turbo.eu/s/CWh

Reply:
Mmhh.. that is some kind of suboptimal 😞 But as far as I can see, this hotel does only exist in France, so a Polish wikipedia article does not make that much sense... Maybe we should just remove it. What you you think @bhousel @matkoniecz ?

Reply:
IMHO linking weird language version is preferable over not linking wikipedia at all. For example some motorway crossings in Poland have German-language wikipedia links, as there is no Polish language article about them.

Reply:
> IMHO linking weird language version is preferable over not linking wikipedia at all. For example some motorway crossings in Poland have German-language wikipedia links, as there is no Polish language article about them.

yeah I agree with @matkoniecz - linking a weird wiki language page is better than linking no language at all 

In fact, the `brand:wikipedia` links are known to be pretty unstable.  However the `brand:wikidata` link is the much more stable id.

We can always go thru later and run a bulk edit on OpenStreetMap to cleanup the tags if better ones become available.  This is only possible once the editor start to assign the tags though - so yeah, imperfect tag is better than no tag 😄